### ISSUE TITLE
Ambiguous one-liner at end of doc

### ISSUE THREAD

**Issue by relaxdiego** [juju-solutions/charms.reactive#212]
This line:

> Note that options of type boolean will set the exit code, while other types will be printed out.

...found at the bottom of [this file](https://github.com/juju-solutions/charms.reactive/blob/master/docs/layer-basic.md) is confusing. What happens when there are more than one boolean options? Is this relevant to all types of handlers or just for the Bash handler? Why does a boolean option determine the exit code while other options do not?

**Comment by johnsca** [comment]
This is only relevant to using the `layer_option` CLI helper.  Specifically, it's saying that calling `layer_option layer opt` will send the value for non-boolean options to stdout, so you would capture it with, e.g.,

```bash
my_var=$(layer_option layer opt)
```
Whereas boolean type options will set the exit code so that you can test them with `if` as you would with, e.g., `grep`:

```bash
if layer_option layer opt; then
  ...
fi
```

**Comment by relaxdiego** [comment]
Thanks, @johnsca! Would it help to append your response to the text in question?

**Comment by johnsca** [comment]
Yeah, I definitely think that clarification would be good to add, though I'm sure it could be worded better for the docs.  :)
